src/VirtoCommerce.Platform.Web/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -107,5 +110,30 @@ public static IServiceCollection AddSecurityServices(this IServiceCollection ser
 
             return services;
         }
+
+
+
+        public static IServiceCollection AddLibraries(this IServiceCollection services)
+        {
+            var providerSnapshot = services.BuildServiceProvider();
+            var assemblyResolver = providerSnapshot.GetRequiredService<IAssemblyResolver>();
+            var hostingEnvironment = providerSnapshot.GetRequiredService<IWebHostEnvironment>();
+            var platformOptions = providerSnapshot.GetService<IOptions<PlatformOptions>>();
+
+            var librariesDir = Path.Combine(hostingEnvironment.ContentRootPath, platformOptions.Value.LibraryPath);
```

Comment:
Library path can be resolved in startup https://github.com/VirtoCommerce/vc-platform/blob/release/3.0.0/src/VirtoCommerce.Platform.Web/Startup.cs#L370